automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -272,4 +274,18 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     testResult.get
   }
 
+  def withNewGoogleBucket[T](googleProject: GoogleProject)(testCode: GcsBucketName => T): T = {
```

Comment:
So this creates a bucket in a google project using the QA service account. I wonder if this method should take a list of emails to grant (read? owner?) access to. For example if a test needs to access the bucket with Ron, would it work?

Reply:
Not a bad idea.  I do need to grant Reader to Ron('s proxy) in my test.

Reply:
I've been thinking about this some more.  I'm not confident that we'll be able to satisfy all testers' intentions here generically in a simple way.  To do this right, we'd have to consider Bucket access, Object access, and Default Object access as well.  At that point I'm thinking that the interface would have to be something like a collection of (AccessType, AccessLevel, GcsEntity) tuples ... which might be useful but it's starting to smell like a yak.

Reply:
OK, makes sense. Can keep it simple for now.